lib/proto.js

```diff
@@ -66,9 +67,8 @@ Object.setPrototypeOf(prototype, EventEmitter.prototype)
 module.exports = prototype
 
 function child (bindings) {
-  const { level } = this
   const serializers = this[serializersSym]
-  const chindings = asChindings(this, bindings)
+  serializers[bindingsSerializerSym] = bindings => bindings
```

Comment:
Can we use a default for this in the prototype chain so we don't have to generate a new closure every time?